api-reference/10 UI Widgets/dxTreeView/1 Configuration/selectNodesRecursive.md

```diff
@@ -5,7 +5,9 @@ default: true
 ---
 ---
 ##### shortDescription
-Specifies whether or not to select nodes recursively.
+Specifies whether all child nodes should be selected when their parent node is. Applies only if the [selectionMode](/api-reference/10%20UI%20Widgets/dxTreeView/1%20Configuration/selectionMode.md '/Documentation/ApiReference/UI_Widgets/dxTreeView/Configuration/#selectionMode') is *"multiple"*.
```

Comment:
Specifies whether all child nodes should be selected when their parent node is. What?